py/server/deephaven/table.py

```diff
@@ -349,7 +388,7 @@ def lazy_update(self, formulas: Union[str, Sequence[str]]) -> Table:
         """
         try:
             formulas = to_sequence(formulas)
-            with auto_locking_ctx(self):
+            with auto_locking_ctx(self), _query_scope_ctx():
```

Comment:
How does globals() work wrt defaults? `def my_method(formulas, context=globals())`? Does it capture `globals()` from the callers perspective, or from the currently defined module?

Reply:
always the module where it is defined